README.md

```diff
@@ -55,4 +55,25 @@ MyChannel.js
       }
     }
 
-Actioncable is good stuff, even if it is in Ruby.
\ No newline at end of file
+Actioncable is good stuff, even if it is in Ruby.
+
+## Connecting from Node.js
+
+`es6-actioncable` will work under Node.js, however you will need to bear the following in mind:
+
+* Your ActionCable Rails server must be bound to a specific IP or `0.0.0.0`, but not localhost. This can be done as follows `rails server -b 0.0.0.0`. See https://twitter.com/mattheworiordan/status/713350750483693568 for an explanation of the issue.
+* You will need to pass the origin to the WebSocket library as Rails will by default reject requests with an invalid origin.  See example below:
```

Comment:
I think before you merge this you should recommend that users use https://www.npmjs.com/package/websocket for a W3C compliant Node.js WebSocket as it has been tested, however then state that any W3C compliant WebSocket can be used. WDYT?